src/components/Table.vue

```diff
@@ -110,7 +110,65 @@
                         >
                         {{ team.points ? team.points : "N/A" }}
                       </td>
+                      <td class="w-full lg:w-auto p-3 text-gray-800 text-center border border-b block lg:table-cell relative lg:static">
+                        <button
+                          @click="expandTeam(team.id)"
+                          class="focus:outline-none"
+                          v-if="
+                            expandedTeamId !== team.id
+                          "
+                        >
+                          <svg
+                            xmlns="http://www.w3.org/2000/svg"
+                            fill="none"
+                            viewBox="0 0 24 24"
+                            stroke="currentColor"
+                            class="w-7 h-7"
+                          >
+                            <path
+                              stroke-linecap="round"
+                              stroke-linejoin="round"
+                              stroke-width="2"
+                              d="M19 9l-7 7-7-7"
+                            />
+                          </svg>
+                        </button>
+                        <button
+                          @click="expandTeam(team.id)"
+                          class="focus:outline-none"
+                          v-if="
+                            expandedTeamId === team.id
+                          "
+                        >
+                          <svg
+                            xmlns="http://www.w3.org/2000/svg"
+                            fill="none"
+                            viewBox="0 0 24 24"
+                            stroke="currentColor"
+                            class="w-7 h-7"
+                          >
+                            <path
+                              stroke-linecap="round"
+                              stroke-linejoin="round"
+                              stroke-width="2"
+                              d="M5 15l7-7 7 7"
+                            />
+                          </svg>
+                        </button>
+                      </td>
                     </tr>
+                    <tr :key="'B' + team.id" v-if="expandedTeamId === team.id">
+                      <td class="col-span-4 md:p-5" colspan="4">
+                        <div class="grid grid-cols-1 md:grid-cols-4 gap-4 mt-2">
```

Comment:
May I suggest the following grid layout ?

`grid-cols-1 md:grid-cols-3 lg:grid-cols-4`

Having 3 columns on a mid-size screen should look a bit better than 4.